### PR TITLE
refactor(legacy-scripting-runner): some tests aren't compiling function source

### PR DESCRIPTION
**Pull request by eliangcs**
Fixes https://github.com/zapier/zapier-platform/pull/210/files#r418914759, where some tests of legacy-scripting-runner aren't properly compiling function source in the app definition. This only fixes the tests and won't affect the library itself.

### CODE REVIEW

**Comment by eliangcs** [comment]
cc @FokkeZB

File: packages/legacy-scripting-runner/test/example-app/index.js [hunk 1]
```diff
@@ -1194,12 +1194,12 @@ const App = {
     },
   },
   // This breaks `applyBeforeMiddleware` which it looks like gets the original definition without `findSourceRequireFunctions` applied?!
```
Comment (FokkeZB) [inline packages/legacy-scripting-runner/test/example-app/index.js:1196]: We can remove this.

File: packages/legacy-scripting-runner/test/example-app/api-key-auth.js [hunk 2]
```diff
@@ -40,13 +32,4 @@ module.exports = {
...
   },
-  beforeRequest: [
-    {
-      source: beforeRequestSource,
-      args: ['request', 'z', 'bundle'],
-    },
-  ],
-  afterResponse: [
-    { source: afterResponseSource, args: ['response', 'z', 'bundle'] },
-  ],
```
Comment (FokkeZB) [inline packages/legacy-scripting-runner/test/example-app/api-key-auth.js:51]: I don't understand why we are and can remove these middlewares for the `custom`, `oauth2` and `session` examples as we do have middleware for them in https://github.com/zapier/zapier-platform/blob/4bae98de1deb3aae3e9c2fa0aede140fad02fde7/packages/legacy-scripting-runner/middleware-factory.js.

File: packages/legacy-scripting-runner/test/example-app/index.js (hunk 1 above)
Comment (eliangcs) [reply packages/legacy-scripting-runner/test/example-app/index.js:1196]: 🤦I'll do that in another PR. Thanks!

File: packages/legacy-scripting-runner/test/example-app/api-key-auth.js (hunk 2 above)
Comment (eliangcs) [reply packages/legacy-scripting-runner/test/example-app/api-key-auth.js:51]: Eventually these auth definitions will be merged to the main app definition. For example, see https://github.com/zapier/zapier-platform/blob/4bae98de1deb3aae3e9c2fa0aede140fad02fde7/packages/legacy-scripting-runner/test/integration-test.js#L14-L16 and https://github.com/zapier/zapier-platform/blob/4bae98de1deb3aae3e9c2fa0aede140fad02fde7/packages/legacy-scripting-runner/test/integration-test.js#L59-L62 

so we only need to write middlewares once in the main app definition.

Comment (FokkeZB) [reply packages/legacy-scripting-runner/test/example-app/api-key-auth.js:51]: Ah, ok. I wonder if developers that look at these examples may get confused and think these middleware are not needed, while this is only due to how we merge in the test app.

Comment (eliangcs) [reply packages/legacy-scripting-runner/test/example-app/api-key-auth.js:51]: > developers that look at these examples may get confused

Hopefully, that doesn't happen. Those "examples" are only for us to run tests, they're never meant to be examples for external developers.

**Pull request merged**